### PR TITLE
update the organization of docs-principles content

### PR DESCRIPTION
**Pull request by stumblefiend**
----
📚 Documentation preview 📚: https://writethedocs-www--2194.org.readthedocs.build/

### CODE REVIEW

File: docs/guide/writing/docs-principles.rst [hunk 1]
```diff
@@ -1,10 +1,9 @@
-Documentation principles
-========================
+Documentation principles for creating great content
```
Comment (plaindocs) [inline docs/guide/writing/docs-principles.rst]: I don't love "content" here at all. It also kinda redundant if we're already saying Documentation.

How about 

> Principles for creating great documentation

File: docs/guide/writing/docs-principles.rst [hunk 2]
```diff
@@ -13,97 +12,28 @@ accountable to producing high-quality code.
...
-Overview, by component
-----------------------
-
-All documentation
-~~~~~~~~~~~~~~~~~
+General documentation principles
+--------------------------------------
 
-In general, documentation should be...
+In general, all documentation should be...
```
Comment (plaindocs) [inline docs/guide/writing/docs-principles.rst]: couple remove General from both of these places imo

File: docs/guide/writing/docs-principles.rst [hunk 3]
```diff
@@ -114,22 +44,32 @@ Participatory
...
 capacity for editorial oversight.
 
 Encourage *everyone* to become a :doc:`documentarian </documentarians>`!
 
-Content should be...
---------------------
+Principles for great content
+-----------------------------
+
+*"Content" is the conceptual information within documentation.*
```
Comment (plaindocs) [inline docs/guide/writing/docs-principles.rst:64]: Do we have a source for this? I'm not sure it's a thing

**Review by plaindocs — COMMENTED** [review]
Love these improvements, and appreciate the improvements you're making.

My objection to content is somewhat personal, but not entirely.

File: docs/guide/writing/docs-principles.rst (hunk 3 above)
Comment (stumblefiend) [reply docs/guide/writing/docs-principles.rst:64]: 🤷

**Review by plaindocs — APPROVED** [review]
I know this isn't your source material, so if you want to merge it in go ahead.

**Pull request merged**